.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 24.4.2
+    hooks:
+      - id: black
+        args:
+          - --line-length=79
+          - --safe
+
+  - repo: https://github.com/PyCQA/flake8
+    rev: 7.0.0
+    hooks:
+      - id: flake8
+        language_version: python3
+        args:
+          - --extend-ignore=E501
+          - --extend-select=B950
+          - --per-file-ignores=./app/api/models.py:F722
```

Comment:
I'm assuming these were copied over from the API repo so I'd suggest removing these `args` as they're specific to the API code